spec/classes/init_spec.rb

```diff
@@ -933,4 +931,43 @@
     end
   end
 
+  describe 'with package_manage set' do
+    let(:facts) { { :osfamily => 'RedHat' } }
+    [true,'true',false,'false'].each do |value|
+      context "to valid #{value} (as #{value.class})" do
+        let(:params) { { :package_manage => value } }
+
+        if value.to_s == 'true'
+          it {
+            should contain_package('ntp').with({
+              'ensure'    => 'present',
+              'noop'      => 'false',
+              'source'    => nil,
+              'adminfile' => nil,
+              'before'    => [ 'File[ntp_conf]', ],
+              'notify'    => [ 'Service[ntp_service]', ], 
+            })
+          }
+        end
+
+        if value.to_s == 'false'
+          it { should_not contain_package('ntp') }
+        end
+      end
+    end
+
+    ['invalid',3,2.42,['array'],a = { 'ha' => 'sh' }].each do |value|
+      context "to invalid #{value} (as #{value.class})" do
+        let(:params) { { :package_manage => value } }
+
+        it do
+          expect {
+            should contain_class('ntp')
+          }.to raise_error(Puppet::Error)
+        end
+
```

Comment:
too picky of me, but since you're rebasing anyhow, could you remove these empty lines